dcor/_energy.py

```diff
@@ -4,9 +4,14 @@
 
 import numpy as np
 
+import enum
+
 from . import distances
 from ._utils import _transform_to_2d
 
+class EstimationStatistic(enum.Enum):
```

Comment:
Maybe change the name to match parameter name?
```suggestion
class StatType(enum.Enum):
```

Reply:
I considered that, but I wanted to add something a bit more specific. "StatType" could refer to any type of statistic, but this makes it evident that these are used to estimate parameters of a distribution.

Reply:
Then maybe the parameter name should change. I think that it is easier for the user if they match.